Inventory.cpp

```diff
@@ -40,9 +40,9 @@ int Inventory::getInventorySlotQuantity(std::string itemName) const
 		if (itemInSlot[i] == itemName)
 		{
 			return quantityInSlot[i];
-			break;
```

Comment:
You can't break after returning. After returning you have already left the function, so the loop you're in doesn't even factor in anymore, so no point in breaking from it.